src/lib.rs

```diff
@@ -18,6 +18,9 @@
 //!
 //! - `upnp-description`: Adds support for accessing the UPnP description of a bridge. See the
 //! [`bridge::Description`] struct for more information.
+//! - `old-api`: minimal effort support for older api versions. Usefull for users of the no longer
```

Comment:
nitpick: "Minimal" instead of "minimal", and "Useful" instead of "Usefull"

Reply:
You just gave me the motivation to keep working on my spelling and grammar plugin for neovim :)